app/models/mentor/application.rb

```diff
@@ -0,0 +1,118 @@
+module Mentor
+  class Application
```

Comment:
Would you mind adding a few sprinkles of (YARD) documentation for future versions of ourselves?